.toolversions

```diff
@@ -0,0 +1,25 @@
+cmake="
+  DeclaredVersion='3.6.0'
+  DownloadFileLinux='cmake-3.6.0-Linux-x86_64.tar.gz'
+  DownloadFileOSX='cmake-3.6.0-Darwin-x86_64.tar.gz'
+  DownloadFileWindows32='cmake-3.6.0-win32-x86.zip'
+  DownloadFileWindows64='cmake-3.6.0-win64-x64.zip'
+  DownloadUrl='https://cmake.org/files/v3.6/'
+  LocalToolFolder='Tools/downloads/cmake'
```

Comment:
I think the two LocalToolFolder properties should be defaulted.  They should be overrideable but we can provide the default value that will work for the majority.